hazelcast/src/main/java/com/hazelcast/util/IterableUtil.java

```diff
@@ -71,4 +72,46 @@ public void remove() {
     public static <T> Iterable<T> nullToEmpty(Iterable<T> iterable) {
         return iterable == null ? Collections.<T>emptyList() : iterable;
     }
+
+    public static <T> Iterable<T> createIterableWithSingleItem(final T item) {
```

Comment:
we can remove it completely since there is already `Collections.singleton*(T value)`
it was used by enterprise